src/core/lib/experiments/experiments.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright 2022 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: tcp_frame_size_tuning
+  description:
+    If set, enables TCP to use RPC size estimation made by higher layers.
+    TCP would not indicate completion of a read operation until a specified
+    number of bytes have been read over the socket.
+    Buffers are also allocated according to estimated RPC sizes.
+  default: false
+  expiry: 2022/09/01
+  test_tags: ["endpoint_test", "core_end2end_test"]
```

Comment:
might be helpful to include the possible test_tags in the README

Reply:
added some text as a comment here